### ISSUE TITLE
Media Library cropper UX/UI

### ISSUE THREAD

**Issue by Katrienvh** [forkcms/forkcms#3108]
## Feature Request

| Q           | A      |
| ----------- | ------ |
| New Feature | yes    |
| RFC         | yes|
| BC Break    | no |

### Summary

There are a lot of things about the cropper I want to say, but the mean message in this issues for me is that we need to look again at the cropper, because I is (looks) really buggy.

1. You have to check the checkbox, at the bottom of the modal where you can upload your media, before you upload media to be able to crop them.
-> It not logic that the checkbox stands under the aria to upload if you have to check it before you upload something

2. You can't cancel the cropper ->  cancel the cropper and continue the upload

3. If you click outside the modal of the cropper, de modal closes but you media still is loading in the uploading aria. It will not finish. -> it should also cancel the cropper and continue the upload.

4. When the image is larger than you window height, you can scroll. But I think it's not user friendly. I should scale the image so it fits in to the screen. I'm also not a big fan of the cropper in a modal.

5. When you want upload multiple images at once. The cropper will open (one after one) for each image. When you're done with the first one and submit (crop button), the modal reopens with the next image. But you do not get a success message that the first crop was successful. Also it would be handy for the user that there is a way he know what he's doing, how much croppers that are coming, ... A cancel button for an image that does not need to be cropped in this case is a must.

6. The styling of the cropper could use some love.

**Comment by sarahmey** [comment]
Yeah let's get some work done on this cropper. It can be a very useful tool but now it just isn't. You should be able to crop after uploading a file. And the modal-thing is indeed just not user-friendly. 

We'll fix this together with rethinking the media-upload flow.
